mteb/model_meta.py

```diff
@@ -51,6 +53,11 @@ def get_loader_name(
     return loader.__name__
 
 
+class EvaluationFunction(str, Enum):
```

Comment:
I believe that's how we've been calling these in AbsTasks?

```suggestion
class ScoringFunction(str, Enum):
```